### DISCUSSION TITLE
Placement of chat - very inconsistent userexperience

### DISCUSSION THREAD

**Discussion by dkmaker** [cline/cline#426] [Feature Requests]
Hi there - im working a lot with VSCode, and i love it and i love Claude Dev for many reasons, one thing that keeps bugging me is the ablility to have a consistent chat experience, wither the windows is on top of the files (VERY anoying) if i undock it changed files come in the new window not the main window when create, splitting the window moves focus away when new files come up, unable to put the window on the buttom half of the screen, preferably its a  tab on the lower part where the terminals, output is, or its stuck on under the files

![image](https://github.com/user-attachments/assets/2a37c4f8-38f5-421e-a3db-936fa5e74df3)

Maybe there is a better way of having a consistent window experience where the folder list is visible and the window stays somehow focused always?

**Comment by tsdevau** [comment]
You can drag Claude-Dev form the activity bar to either of the two option places.  If I understand you correctly?

### As a Tab in Bottom Panel

<img width="1296" alt="Screenshot 2024-10-02 at 08 39 40" src="https://github.com/user-attachments/assets/e89762a5-44fe-4b2a-9737-a80e496fad4e">

### As a Section in Explorer Sidebar

<img width="1296" alt="Screenshot 2024-10-02 at 08 39 00" src="https://github.com/user-attachments/assets/4a6c058b-f425-4b11-9cca-0316774a5033">

**Comment by dkmaker** [comment]
Ahh.. Got it my issue was i unpopped so it was not able to fit into the screen
